cas-server-support-osf/src/main/java/org/pac4j/oauth/client/OrcidClient.java

```diff
@@ -167,20 +168,26 @@ public void setMember(final Boolean member) {
     }
 
     /**
-     * Extract user profile.
+     * Extract the user profile from the profile response.
+     *
+     * This method replaces itself and here are the customizations:
      *
-     * @param body the body
+     * 1. Set the raw ORCiD ID as the profile ID
+     * 2. Build the profile using normalized attributes so they can be successfully released to OSF
+     *
+     * @param body the response body
      * @return the profile
      */
     @Override
     protected OrcidProfile extractUserProfile(final String body) {
+
         final OrcidProfile profile = new OrcidProfile();
         profile.setId(XmlHelper.get(body, OrcidAttributesDefinition.ORCID));
+
         for(final String attribute : OAuthAttributesDefinitions.orcidDefinition.getAllAttributes()) {
             final String value = XmlHelper.get(body, attribute);
             switch (attribute) {
                 case OrcidAttributesDefinition.NORMALIZED_FAMILY_NAME:
-                    break;
```

Comment:
Why was this removed?

Reply:
I had the same question when I saw the style warning for this `break;` statement. I went on and searched. It turns out that for Java `switch` statement, if the same action (here just the `break;`) is used for multiple cases, I should group the cases and just write the action once.